<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Evorl's passgen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/pngegg.ico</normaloff>:/icons/pngegg.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
    color: black;
    font-family: Verdana;
    font-size: 16pt;
    margin: 10px;
}

QPushButton {
    border: 2px solid gray;
    border-radius: 5px;
}

QPushButton#btn_lower, #btn_upper, #btn_special,#btn_digits  {
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    border-color: lightseagreen;
}

QPushButton:pressed {
    border: 4px solid lightseagreen;
    border-radius: 5px;
}

QPushButton:checked {
    border-color: lightseagreen;
    background-color: rgb(21, 110, 106);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: white;
    color: black;
    font-family: Verdana;
    font-size: 16pt;
    margin: 10px;
}

QPushButton {
    border: 2px solid gray;
    border-radius: 5px;
}

QPushButton#btn_lower, #btn_upper, #btn_special,#btn_digits  {
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    border-color: lightseagreen;
}

QPushButton:pressed {
    border: 4px solid lightseagreen;
    border-radius: 5px;
}

QPushButton:checked {
    border-color: lightseagreen;
    background-color: rgb(21, 110, 106);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="icon">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="rsc.qrc">
        <normaloff>:/icons/pngeggg.ico</normaloff>:/icons/pngeggg.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_pass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked {
    border-color: lightseagreen;
    background-color: rgb(21, 110, 106);
}

QFrame:hover {
    border: 2px solid lightseagreen;
    border-radius: 5px;
    margin-right: 0;
}
QFrame {
    border: 2px solid gray;
	border-radius: 5px;
	margin-right: 0;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="line_pass">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: none;
    margin: 0;
    font-size: 20pt;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_visi">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: none;
    margin: 0;
    background-color: transparent;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rsc.qrc">
             <normaloff>:/icons/visibility_black_24dp.svg</normaloff>
             <normalon>:/icons/visibility_off_black_24dp.svg</normalon>:/icons/visibility_black_24dp.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_refresh">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    margin-left: 0;
    margin-right: 0;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icons/refresh_black_24dp.svg</normaloff>:/icons/refresh_black_24dp.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>52</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_copy">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    margin-left: 0;
    margin-right: 10;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icons/content_copy_black_24dp.svg</normaloff>
          <normalon>:/icons/content_copy_white_24dp.svg</normalon>:/icons/content_copy_black_24dp.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>52</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+C</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="text_strengt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="text_entropy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_lenght">
      <item>
       <widget class="QSlider" name="slider_lenght">
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    background-color: transparent;
    height: 5px;
}

QSlider::sub-page:horizontal {
    background-color:  rgb(21, 110, 106);
}

QSlider::add-page:horizontal {
    background-color:  gray;
}

QSlider::handle:horizontal {
    background-color:  black;
    width: 10px;
    border-radius: 3px;
    margin-top: -10px;
    margin-bottom: -10px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinbox_lenght">
        <property name="styleSheet">
         <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 5px;
    background: transparent;
    padding: 2px;
}

QSpinBox:hover {
    border-color: lightseagreen;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_char">
      <item>
       <widget class="QPushButton" name="btn_lower">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>a-z</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_upper">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>A-Z</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_digits">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>0-9</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_special">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>*%@</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
